src/readii/negative_controls.py

```diff
@@ -461,12 +529,28 @@ def applyNegativeControl(nc_type: str, baseImage: sitk.Image, baseROI: sitk.Imag
     -------
     sitk.Image
         The output image with the negative control applied
-     """
-
+        
+    Raises
+    ------
+    ValueError
+        If the nc_type is not a valid negative control type
+    """
+    
     if nc_type == "randomized_full":
         # Make negative control version of ctImage (randomized pixel size)
         return makeRandomImage(baseImage)
-    elif nc_type == "randomized_roi":
+    elif nc_type == "shuffled_full":
```

Comment:
These two elif clauses moved here because they don't require the baseROI or roiLabel